### DISCUSSION TITLE
在macOS上使用Homebrew安裝並管理Git和Python

### DISCUSSION THREAD

**Discussion by Yunhans** [1101-GS4719-C/Python-Community-GS4719-C#682] [Show and tell]
Homebrew 是一個非常方便在macOS上安裝及管理更新的套件（windows沒有）
雖然在Git以及Python官網也可以個別下載
不過Homebrew有一個很酷的功能就是把套件「一次全部更新」
酷吧

1. 安裝Homebrew
在終端機（terminal）內輸入
`/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"`
就是那麼方便

2. 安裝Git
在終端機（terminal）內輸入
`brew install git`

3. 安裝Python
在終端機（terminal）內輸入
`brew install python`
如果要指定Python版本則在Python後面加上@再加上版本號，例如想安裝Python3.7就輸入
`brew install python@3.7`

4. 查看已安裝套件
在終端機（terminal）內輸入
`brew list`
便可以查看所有用brew安裝的套件（就像pip3 list)

5. 更新套件
在終端機（terminal）內輸入
`brew upgrade && brew update`
等待終端機更新完成就行了，非常簡單

[Homebrew 官網](https://brew.sh/index_zh-tw)

**Comment by a45320614** [comment]
真的很酷呢! 不知道Windows之後會不會類似的套件

**Comment by danwan1027** [comment]
能針對 mac 用戶做出整理真的很實用

**Comment by dingxue9080** [comment]
整理好的mac套件真的實用！

**Comment by Chiway1229** [comment]
在windows沒有這個套件實在是太可惜了
希望有朝一日也能有類似的套件
方便電腦一次更新

**Comment by Evan0979** [comment]
感謝分享,身為一個MAC使用者的我覺得這篇文章必須置頂

**Comment by williamyen0623** [comment]
brew真的超方便！安裝沒煩惱～

**Comment by Doreen331** [comment]
mac在許多環境安裝上真的挺不友善的，但這個套件真的很方便！也希望mac可以讓使用者在安裝環境上更方便一點，不然真的要跳回windows了😢

**Comment by manipulateed** [comment]
居然有這麼方便的整合套件，不走也寫得非常仔細，可惜我是用windows沒法體驗。

**Comment by evanchou1204** [comment]
感覺在mac上下載什麼東西都限制超多，有了這個真的蠻方便的，不然寫個python都會卡爛超不爽。

**Comment by Pietra1226** [comment]
我之前為了寫一個特定的code從MAC轉來windows，後來就回不去了，方便又可以打遊戲，太舒服了
